src/legacy/core_plugins/kibana/public/discover/np_ready/angular/discover.js

```diff
@@ -806,7 +806,7 @@ function discoverController(
       .then(onResults)
       .catch(error => {
         // If the request was aborted then no need to surface this error in the UI
-        if (error instanceof Error && error.name === 'AbortError') return;
+        if (error instanceof Error && error.message === 'The user aborted a request.') return;
```

Comment:
Yeah, unfortunately I'd much prefer using the `name` but obviously won't work until https://github.com/elastic/kibana/issues/51406.

Reply:
I actually fixed this, unaware that this was an open issue.
Especially as there are more places where this was a problem (from @stacey-gammon review)
Hope @elastic/kibana-platform approves.